rsocket/RSocketStats.h

```diff
@@ -38,7 +40,7 @@ class RSocketStats {
   virtual void streamBufferChanged(
       int64_t framesCountDelta,
       int64_t dataSizeDelta) = 0;
-
+  virtual void resumeFailedNoState() = 0;
```

Comment:
nit: we might rename this function as resumeFailed() and create an enum of possible errors and pass it in this function. Or we might pass in a folly::exception_wrapper into this function. So we can let the party which implements this function to be able to know which kind of error has happened.

Reply:
once we have more than one case of resumeFailed, I would go with Fuat's suggestion. Right now I can see only once  of resumeFailed.